### PR TITLE
Update the OGM Getting Started Guide

### PR DESCRIPTION
**Pull request by Sanne**
It seems that the website update which we had scheduled for the release got lost ?!

Here is a fresh proposal.

### CODE REVIEW

**Comment by Sanne** [comment]
Unless someone overwrites it, it should be visible on:
- http://staging.hibernate.org/ogm/documentation/getting-started/

@gsmet @gunnarmorling @DavideD [sorry for the mentions, I'm not sure you're all listening for PRs on this repo]

**Comment by Sanne** [comment]
This was tracked as https://hibernate.atlassian.net/browse/WEBSITE-443
Updating commit message..

**Comment by gsmet** [comment]
Looks good to me apart from a minor typo I'll fix on merging.

**Comment by gunnarmorling** [comment]
To be sure, that's a plain copy from the ref guide, right?

**Comment by Sanne** [comment]
@gunnarmorling that's right, with the exception of the closing paragraph pointing to the docs rather than the wording "in the next chapter [...]" ;)

File: ogm/documentation/getting-started.adoc [hunk 1]
```diff
@@ -236,33 +238,18 @@ tm.commit();
...
 What have we seen?
 
 * Hibernate OGM is a JPA implementation
   and is used as such both for mapping and in API usage
 * It is configured as a specific JPA provider:
-  [classname]+org.hibernate.ogm.jpa.HibernateOgmPersistence+
+  `org.hibernate.ogm.jpa.HibernateOgmPersistence`
 
-Let's explore more in the next chapters.
+To learn more about Hibernate OGM an all the NoSQL technologies it supports, see the http://hibernate.org/ogm/documentation/[Reference Documentation].
```
Comment (Sanne) [inline ogm/documentation/getting-started.adoc:254]: This last line is the difference vs the example in the reference docs @gunnarmorling

**Comment by gsmet** [comment]
Merged! Thanks.

File: ogm/documentation/getting-started.adoc (hunk 1 above)
Comment (gunnarmorling) [inline ogm/documentation/getting-started.adoc:254]: Cool, thanks!

**Pull request closed without merging**